libs/ui-watt/src/lib/components/autocomplete/autocomplete.component.ts

```diff
@@ -23,7 +23,7 @@ import { FormControl } from '@angular/forms';
   styleUrls: ['./autocomplete.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AutocompleteComponent {
+export class WattAutocompleteComponent {
   /**
    * @ignore
```

Comment:
Yes, it looks like [it is](https://compodoc.app/guides/jsdoc-tags.html). Do you prefer using `@internal` instead?

Reply:
It's the first time I see `@ignore`.

`@internal` is used by Angular and supported by the TypeScript compiler and Typedoc.

Reply:
@dzhavat Do you create an issue for this? Let's fix it before we have @ignore to many places, if @internal is the way to go :)

Reply:
@frontend-specialisten I will create a PR shortly after daily, so I don't forget about it.

Reply:
@dzhavat Perfect.

Reply:
@LayZeeDK The main reason for using `@ignore` is because of Storybook. When using this tag, Storybook will ignore the property and not render it in the props table.

Replacing `@ignore` with `@internal`, however, does not work the same. Just tried it on a local branch. The property marked with `@internal` is rendered:

![image](https://user-images.githubusercontent.com/1096332/135987213-1f1a6f68-9e6e-4df5-81b9-f5cb0ad61a5d.png)

If we want to use the `@internal` tag anyway, there's another way to ignore a property in Storybook by using the following: 

```
export default {
  component: YourComponent,
  title:'My Story',
  argTypes:{
    // foo is the property we want to remove from the UI
    foo:{
      table:{
        disable:true
      }
    }
  }
};
```

However, this adds extra complexity and more things to remember and maintain. That being said, I prefer keeping the `@ignore` flag. What do you think?